sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/DocumentModelAdministrationLiveTests.cs

```diff
@@ -176,170 +176,237 @@ public void BuildModelError()
         [RecordedTest]
         [TestCase(true)]
         [TestCase(false)]
-        public async Task CopyModelTo(bool useTokenCredential)
+        public async Task CopyDocumentModelTo(bool useTokenCredential)
```

Comment:
Improving the validation of our "copy" test. Originally we were just validating major properties and doing some sanity checks but now we're validating all properties returned by the service.